components/common/listingCart/SuccessfulListingBody.vue

```diff
@@ -28,10 +28,11 @@ const { accountId } = useAuth()
 const mediaItems = computed<ItemMedia[]>(() =>
   props.items.map((item) => ({
     id: item.id,
-    image: item.meta?.image as string,
+    image: (item.animationUrl?.url || item.meta?.image) as string,
```

Comment:
animationUrl at this point is full gen art no ?

Reply:
yes, `animationUrl` is not a static image

https://image-beta.w.kodadot.xyz/ipfs/bafybeicdv7iu5racra3vsvg2k2uvp2jxvb534h67f6lqdwidhmvzg2tjou/?hash=0xed3dcf74fda65f4797e006c2101350fd7703d253fae8c26c932f6b10f96d46e1 

side note: I was thinking if we should use static images any time we have more than x amount(3 maybe) of items , 
that would improve performance , we've see how some drops do consume a lot of resources.

Reply:
Yes, thats why this is not a correct fix :)

Reply:
fixed